docs/source/examples/example_doc_generator.py

```diff
@@ -14,18 +14,22 @@
 
 Generate an rst file, then open the app and take a snapshot.
 
+Note: Many examples have dependencies outside of the normal Enaml dependencies.
+These will need to be installed for the snapshots to be generated.
+
 """
 from __future__ import print_function
 import os
+import re
 import shutil
 
 from atom.api import Atom, Unicode, Value
 import enaml
 from enaml.qt.qt_application import QtApplication
 from enaml.qt.QtWidgets import QApplication
 from enaml.qt.QtTest import QTest
-from enaml.application import timed_call
 
+import sys # TODo: Remove
 
```

Comment:
Should have an extra newline here for  and between the save_snapshot_of_module, extract_docstring and clean_docstring functions below to make PEP-8 linters happy.